packages/core/dist/components/MonthPicker/MonthPicker.js

```diff
@@ -90,8 +90,6 @@ var MonthPicker = exports.MonthPicker = function (_React$PureComponent) {
       // Since this isn't a controlled component, we need a way
       // to track when the value has changed.
       _this.state = {
```

Comment:
Totally fine that you've been checking these in, but in case it's helpful: We run the build process right before the release process, so if you want, you can skip that step for PR's.

Reply:
I'd like to skip having these rendered files in PRs to be honest. I'll avoid adding them in the future.